src/cpp/session/resources/grid/datatables/js/jquery.dataTables.min.js

```diff
@@ -1 +1,4 @@
(){return this.context.length?this.context[0].oLoadedState:null}),e("state.save()",function(){return this.iterator("table",function(t){ne(t)})}),F.versionCheck=F.fnVersionCheck=function(t){for(var e,a,n=F.version.split("."),r=t.split("."),o=0,i=r.length;o<i;o++)if((e=parseInt(n[o],10)||0)!==(a=parseInt(r[o],10)||0))return a<e;return!0},F.isDataTable=F.fnIsDataTable=function(t){var a=U(t).get(0),n=!1;return U.each(F.settings,function(t,e){e.nTable!==a&&e.nScrollHead!==a&&e.nScrollFoot!==a||(n=!0)}),n},F.tables=F.fnTables=function(e){return U.map(F.settings,function(t){if(!e||e&&U(t.nTable).is(":visible"))return t.nTable})},F.util={throttle:Bt,escapeRegex:yt},F.camelToHungarian=L,e("$()",function(t,e){var a=this.rows(e).nodes(),n=U(a);return U([].concat(n.filter(t).toArray(),n.find(t).toArray()))}),U.each(["on","one","off"],function(t,a){e(a+"()",function(){var t=Array.prototype.slice.call(arguments);t[0].match(/\.dt\b/)||(t[0]+=".dt");var e=U(this.tables().nodes());return e[a].apply(e,t),this})}),e("clear()",function(){return this.iterator("table",function(t){Q(t)})}),e("settings()",function(){return new v(this.context,this.context)}),e("data()",function(){return this.iterator("table",function(t){return S(t.aoData,"_aData")}).flatten()}),e("destroy()",function(h){return h=h||!1,this.iterator("table",function(e){var a,t=e.nTableWrapper.parentNode,n=e.oClasses,r=e.nTable,o=e.nTBody,i=e.nTHead,s=e.nTFoot,l=U(r),u=U(o),c=U(e.nTableWrapper),f=U.map(e.aoData,function(t){return t.nTr});e.bDestroying=!0,fe(e,"aoDestroyCallback","destroy",[e]),h||new v(e).columns().visible(!0),c.unbind(".DT").find(":not(tbody *)").unbind(".DT"),U(We).unbind(".DT-"+e.sInstance),r!=i.parentNode&&(l.children("thead").detach(),l.append(i)),s&&r!=s.parentNode&&(l.children("tfoot").detach(),l.append(s)),l.detach(),c.detach(),e.aaSorting=[],e.aaSortingFixed=[],ee(e),U(f).removeClass(e.asStripeClasses.join(" ")),U("th, td",i).removeClass(n.sSortable+" "+n.sSortableAsc+" "+n.sSortableDesc+" "+n.sSortableNone),e.bJUI&&(U("th span."+n.sSortIcon+", td span."+n.sSortIcon,i).detach(),U("th, td",i).each(function(){var t=U("div."+n.sSortJUIWrapper,this);U(this).append(t.contents()),t.detach()})),!h&&t&&t.insertBefore(r,e.nTableReinsertBefore),u.children().detach(),u.append(f),l.css("width",e.sDestroyWidth).removeClass(n.sTable),(a=e.asDestroyStripes.length)&&u.children().each(function(t){U(this).addClass(e.asDestroyStripes[t%a])});var d=U.inArray(e,F.settings);-1!==d&&F.settings.splice(d,1)})}),F.version="1.10.4",F.settings=[],F.models={},F.models.oSearch={bCaseInsensitive:!0,sSearch:"",bRegex:!1,bSmart:!0},F.models.oRow={nTr:null,anCells:null,_aData:[],_aSortData:null,_aFilterData:null,_sFilterRow:null,_sRowStripe:"",src:null},F.models.oColumn={idx:null,aDataSort:null,asSorting:null,bSearchable:null,bSortable:null,bVisible:null,_sManualType:null,_bAttrSrc:!1,fnCreatedCell:null,fnGetData:null,fnSetData:null,mData:null,mRender:null,nTh:null,nTf:null,sClass:null,sContentPadding:null,sDefaul
```

Comment:
Since this file is minified I can't tell what's changed here. Did you make an actual change or is this an artifact of some kind?

Reply:
The change is visible in `jquery.dataTables.js` that lives alongside the minified version.

Reply:
Ah, right! 🤦‍♂